bound/kt/src/main/kotlin/web5/sdk/crypto/keys/InMemoryKeyManager.kt

```diff
@@ -0,0 +1,43 @@
+package web5.sdk.crypto.keys
+
+import web5.sdk.crypto.signers.Ed25519Signer
+import web5.sdk.rust.InMemoryKeyManager as RustCoreInMemoryKeyManager
+
+/**
+ * A class for managing cryptographic keys in-memory.
+ */
+class InMemoryKeyManager : KeyManager {
+    private val rustCoreKeyManager = RustCoreInMemoryKeyManager()
+
+    /**
+     * Generates new key material and returns the public key represented as a Jwk.
+     *
+     * @return Jwk the public key represented as a Jwk.
+     */
+    override fun generateKeyMaterial(): Jwk {
+        val jwkData = rustCoreKeyManager.generateKeyMaterial()
+        return Jwk.fromBinding(jwkData)
```

Comment:
Maybe `fromRustCore()` as to be consistent with the `as RustCoreInMemoryKeyManager`?

Reply:
Yup lets do it ✅